### PR TITLE
update packages, node, and script in html

### PR DESCRIPTION
**Pull request by danielleadams**
Updates the packages, Node version, and some slight code to work. (Will be accompanied by updated docs)

### CODE REVIEW

**Comment by danielleadams** [comment]
@jmorrell Not sure if you saw this, but it could use a review!

File: index.html [hunk 1]
```diff
@@ -1,14 +1,17 @@
 <html>
-  <body>
-    <p id='server-time'></p>
+  <head>
     <script>
-      var HOST = location.origin.replace(/^http/, 'ws')
-      var ws = new WebSocket(HOST);
-      var el = document.getElementById('server-time');
+    var HOST = location.origin.replace(/^http/, 'ws')
```
Comment (jmorrell) [inline index.html]: While you're here, you might want to update to modern JS with `let` and arrow functions

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "websockets-vanilla",
   "version": "1.0.0",
   "engines": {
-    "node": "6.2.0"
+    "node": "13.1.x"
```
Comment (jmorrell) [inline package.json]: I tried to stick with the LTS major versions (`10.x`, `12.x`) unless there is some feature from a current version (`13.x`) that's necessary

File: package.json [hunk 3]
```diff
@@ -13,9 +13,9 @@
...
   "license": "ISC",
   "dependencies": {
-    "bufferutil": "1.2.1",
-    "express": "4.13.4",
-    "utf-8-validate": "1.2.1",
-    "ws": "1.1.0"
+    "bufferutil": "^4.0.1",
+    "express": "^4.17.1",
+    "utf-8-validate": "^5.0.2",
+    "ws": "^7.2.0"
```
Comment (jmorrell) [inline package.json:19]: These are some impressive major version jumps :O

**Review by jmorrell — APPROVED** [review]
Looks good 👍

File: package.json (hunk 3 above)
Comment (danielleadams) [reply package.json:19]: I know!

**Pull request merged**